apps/dashboard/src/tw-components/nft-media.tsx

```diff
@@ -18,10 +18,12 @@ export const NFTMediaWithEmptyState: React.FC<{
   if (!(props.metadata.image || props.metadata.animation_url)) {
     return (
       <div
+        style={{
+          width: props.width,
+          height: props.height,
+        }}
         className={cn(
           "overflow-hidden rounded-xl object-contain flex-shrink-0 border grid place-items-center",
```

Comment:
nit: On pages router - we can not just add `border` - we also need to include `border-border` to properly override the chakra's border color